Jenkinsfile

```diff
@@ -92,6 +97,30 @@ String getLabel() {
   return result
 }
 
+boolean getAllFuncTests() {
+  def paramVal = null
+  try {
+    paramVal = params.allFuncTests
+  } catch (e1) {
+    // workaround for https://issues.jenkins-ci.org/browse/JENKINS-38813
+    echo '[WARNING] unable to access `params`'
+    echo getStackTrace(e1)
+    try {
+      paramVal = allFuncTests
+    } catch (e2) {
+      echo '[WARNING] unable to access `allFuncTests`'
+      echo getStackTrace(e2)
```

Comment:
Would all this logging happen in the default case where the parameter is not specified?

If so it is a bit noisy for a non-error case. If it only happens under error conditions I have no issue with it.